.github/workflows/rust.yml

```diff
@@ -72,7 +72,7 @@ jobs:
       with:
         name: cargo-semver-checks
         version: 0.27.0
-    - run: cargo semver-checks
+    - run: cargo semver-checks --exclude soroban-simulation
```

Comment:
The simulation crate isn't published yet and the check fails trying to pull it from crates.io. Excluding it is required because semver-checks requires the crate to be published first. See:
- https://github.com/obi1kenobi/cargo-semver-checks/issues/481